spec/models/period_contribution_calculator_spec.rb

```diff
@@ -24,6 +25,10 @@
       let(:employee_percent) { 1 }
       let(:employer_percent) { 1 }
 
+      it 'knows which period its calculating contributions' do
+        expect(period_contribution.name).to eq 'some period'
```

Comment:
Or parentheses if you're using both `(` and `)` (parenthesis is singular, not plural, and is more commonly used for the grammatical structure of putting an explanation in parentheses … like this :) )